packages/dd-trace/src/config-helper.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+// TODO: Add jira tickets
+
+const { debuglog, deprecate } = require('util')
+const { supportedConfigurations, aliases, deprecations } = require('./supported-configurations.json')
+
+const aliasToCanonical = {}
+for (const canonical of Object.keys(aliases)) {
+  for (const alias of aliases[canonical]) {
+    if (aliasToCanonical[alias]) {
+      throw new Error(`The alias ${alias} is already used for ${aliasToCanonical[alias]}.`)
+    }
+    aliasToCanonical[alias] = canonical
+  }
+}
+
+// TODO: Consider to join deprecations with aliases by just making those entries an object.
+const deprecationMethods = {}
+for (const deprecation of Object.keys(deprecations)) {
+  deprecationMethods[deprecation] = deprecate(
+    () => {},
+    `The environment variable ${deprecation} is deprecated. Please use ${aliasToCanonical[deprecation]} instead.`,
+    `DATADOG_${deprecation}`
```

Comment:
I wonder if this can easily be confused with an actual environment variable..... could this somehow include the word `DEP` so it's similar to the Node.js deprecations? Technically there's no rule that I know of saying they have the use the same `code` between reboots, so it could just be a sequential number also... e.g. `DD_DEP0001` etc (but I'd prefer if they never changed, which would make googling easier)

Reply:
We can use what ever prefix we like. I used this, since it seemed most intuitive for me and the sequential one change over time. This implementation keeps it consistent. I would like that.

Reply:
Yeah I agree... just don't like how much these actually look like environment variables as it might confuse users. But it's not a deal breaker to me, so feel free to keep it as is.

Reply:
They are logged as error code, so it should at least be clear as soon as we receive the deprecation warning :)
<img width="548" alt="image" src="https://github.com/user-attachments/assets/885b9266-583a-4da0-ac24-a20dfef70606" />